documentation/src/docs/metrics.md

```diff
@@ -116,204 +101,158 @@ public final class ServiceStarter {
+operations of createSegment. We will get the time of every createSegment operation happened, how long each operation
+takes, and other numbers computed based on them.
 
 
-### 2.2. Example for Dynamic Gauge and OpStatsLogger(Histogram) 
-This is an example from io.pravega.controller.store.stream.AbstractStreamMetadataStore. In this class, we report a Dynamic Gauge which represents the open transactions and  one histogram (CREATE_STREAM).
+### 2.2. Example for Dynamic Gauge 
+This is an example from io.pravega.controller.metrics.StreamMetrics. In this class, we report 
+a dynamic gauge which represents the open transactions of a stream.
 ```java
-public abstract class AbstractStreamMetadataStore implements StreamMetadataStore  {
+public final class StreamMetrics extends AbstractControllerMetrics implements AutoCloseable {
     ...
-    private static final OpStatsLogger CREATE_STREAM = STATS_LOGGER.createStats(MetricsNames.CREATE_STREAM); // get stats logger from MetricsProvider
-    private DynamicLogger dynamicLogger = MetricsProvider.getDynamicLogger(); // get dynamic logger from MetricsProvider
+    static final DynamicLogger DYNAMIC_LOGGER = MetricsProvider.getDynamicLogger();
        
     ...
-    @Override
-        public CompletableFuture<CreateStreamResponse> createStream(final String scope,
-                                                       final String name,
-                                                       final StreamConfiguration configuration,
-                                                       final long createTimestamp,
-                                                       final OperationContext context,
-                                                       final Executor executor) {
-            return withCompletion(getStream(scope, name, context).create(configuration, createTimestamp), executor)
-                    .thenApply(result -> {
-                        if (result.getStatus().equals(CreateStreamResponse.CreateStatus.NEW)) {
-                            CREATE_STREAM.reportSuccessValue(1); // Report success event for histogram metric CREATE_STREAM
-                            dynamicLogger.reportGaugeValue(nameFromStream(OPEN_TRANSACTIONS, scope, name), 0); // Report gauge value for Dynamic Gauge metric OPEN_TRANSACTIONS 
-                        }
-    
-                        return result;
-                    });
-        }
+    public void createStream(String scope, String streamName, int minNumSegments, Duration latency) {
+        DYNAMIC_LOGGER.reportGaugeValue(OPEN_TRANSACTIONS, 0, streamTags(scope, streamName));
+    }
     ...
  }
 ```
 
 ### 2.3 Example for Dynamic Meter
-This is an example from io.pravega.segmentstore.server.SegmentStoreMetrics. In this class, we report a Dynamic Meter which represents the segments created.
+This is an example from io.pravega.segmentstore.server.SegmentStoreMetrics. In this class, we report a Dynamic Meter 
```

Comment:
Sometimes you refer to the classes in caps (e.h., `Dynamic Meter`) and other you don't. Please, look for a consistent naming.